docs/rules/prefer-power-assert.md

```diff
@@ -0,0 +1,33 @@
+# Only allow use of the assertions that have no power-assert alternative
+
+- `t.ok()` __(You can do most things with this one)__
```

Comment:
Linkify `most things` to https://github.com/sindresorhus/ava#enhanced-asserts (or another source if you have a better one)